src/commands/up.rs

```diff
@@ -213,6 +215,8 @@ impl UpCommand {
             )?;
         }
 
+        self.update_locked_app(&mut locked_app);
```

Comment:
This moved up because the borrow of the trigger type strings (in the next stanza) now lasts longer, so I have to get the mutable borrow out the way first, or the compiler gets mad.